src/extensions/extension-loader.ts

```diff
@@ -86,6 +86,10 @@ export class ExtensionLoader extends Singleton {
     return extensions;
   }
 
+  @computed get userExtensionList() {
```

Comment:
Why do we need `computed` for `computed`? :)
Better to change to `@computed get userExtensions(): InstalledExtension[]` everywhere.

Reply:
We have usage of `userExtensions.get(id)`, `userExtensions.has(id)` in `extensions.tsx`. Not sure if we should replace this with array-like behavior.

Reply:
How about extra method `public getById(id: string): LensExtension`?

Reply:
One thing touches another, there's mix of `extensions`, `userExtensions`, instances etc. There's need to refactor `extension-loader.ts` carefully. For this PR i've removed this `userExtensionList()` method for now.